src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -28,23 +28,29 @@ import { UserQuery } from '../../../shared/user/user.query';
 import { UserService } from '../../../shared/user/user.service';
 import { TokenUser } from './../../../shared/swagger/model/tokenUser';
 import { AccountsService } from './accounts.service';
+import { User } from '../../../shared/swagger/model/user';
+import { ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'app-accounts-external',
   templateUrl: './accounts.component.html',
-  styleUrls: ['./accounts.component.css'],
+  styleUrls: ['./accounts.component.scss'],
+  encapsulation: ViewEncapsulation.None,
 })
 export class AccountsExternalComponent implements OnInit, OnDestroy {
   public dsServerURI: any;
   public orcidId$: Observable<string>;
   public TokenSource = TokenSource;
+  public user: User;
   Dockstore = Dockstore;
   // TODO: Uncomment section when GitLab is enabled
   accountsInfo: Array<any> = [
```

Comment:
This was like this, so no need to change for this PR, but we should ideally move away from `any` and define a type.